dirs/dirs.go

```diff
@@ -125,6 +125,10 @@ var (
 	FeaturesDir string
 
 	RunMnt string
```

Comment:
so we didn't have the bootloader directories defined, here or /boot.  I think what we likely want to do is move these definitions over the boot, at which point boot using them is a bit less controversial.

sysconfig has also usages, but that one should really take a target rootdir

Reply:
we''ll need a tiny bit of help from dirs, to do this in a not annoying way

Reply:
this is all changed now